app/js/components/visitsAndEncounters.jsx

```diff
@@ -151,34 +152,52 @@ export class VisitsAndEncounters extends React.Component {
         output.push(data)
         data = []
       })
+    });
+    this.state.encountersWithNoVisits.map((encounter) => {
+      data["location"] = encounter.location.display
+      data["date"] = encounter.encounterDatetime
+      data["encounter"] = encounter.display
+      data["encounterUuid"] = encounter.uuid
+      output.push(data)
+      data = []
     })
-    return output
-  }
 
-  render() {
-    const columns = [{
-        id: 'type',
-        Header: 'Type',
-        accessor: rowProps => <a>{rowProps.encounter.slice(0, -10)}</a>,
-      },
-      {
-        id: 'location',
-        Header: 'Location',
-        accessor: rowProps => rowProps.location,
-      },
-      {
-        id: 'date',
-        Header: 'Date',
-        accessor: rowProps => rowProps.date.split("T")[0],
-      },
-      {
-        id: 'visit',
-        Header: 'Visit',
-        accessor: rowProps => <a>{rowProps.visit}</a>,
-      }
-    ];
-    return(
-      <div>
+    function compareEncountersbyDate(firstEncounter, SecondEncounter) {
+      if(firstEncounter.date > SecondEncounter.date)
+        return -1;
+      if(firstEncounter.date < SecondEncounter.date)
+        return 1;
+      return 0;
+    }
+
+  return output.sort(compareEncountersbyDate)
+}
+
+render() {
+  console.log("data", this.getAllEncounters())
```

Comment:
Remove this console statement

Reply:
Thanks for noticing that, i have removed it